docs/CONTRIBUTING.md

```diff
@@ -150,7 +150,7 @@ follow the following format.
 `<TYPE>(<optional SCOPE>): <conditionally BREAKING CHANGE:> <description>`
 
 Want help with your pull request title? We have a
-[tool to help](https://atlantis.getjobber.com/pull-request-name-generator).
```

Comment:
might be best to keep this path absolute since you can view this through GitHub https://github.com/GetJobber/atlantis/blob/7119049f3d63e633bc45202bd1fdaae1aed172e1/docs/CONTRIBUTING.md#opening-a-pull-request